packages/hash/frontend/src/components/GlideGlid/glide-grid.tsx

```diff
@@ -39,6 +50,14 @@ export const GlideGrid = (props: DataEditorProps) => {
       smoothScrollY
       getCellsForSelection
       {...props}
+      /**
+       * icons defined via `headerIcons` are avaiable to be drawn using
+       * glide-grid's `spriteManager.drawSprite`,
+       * which will be used to draw svg icons inside custom cells
+       */
+      headerIcons={customGridIcons}
     />
   );
 };
+
+export const GlideGrid = forwardRef(_GlideGrid);
```

Comment:
This will lead to the component being called `_GlideGrid` in dev tools. The right approach is either:

(this is confirmed to work)

```
const GlideGrid = (…) => { … };

const GlideGridForwardRef = forwardRef(GlideGrid);

export { GlideGridForwardRef as GlideGrid };
```

or this, which supposedly works but I haven't verified

```
export const GlideGrid = forwardRef((...) => { … }); 
```

Reply:
Is this an actual issue to worry about? I feel like having `_GlideGrid` is also fine. We can still search "GlideGrid" to see the component in dev tools. Do we really want to add that extra line for removing the underscore from devtools? I didn't get the motivation here.


![Screen Shot 2022-10-26 at 10 26 17](https://user-images.githubusercontent.com/39553853/197961607-f08cad24-96f0-40fb-a287-34a52ccecae7.png)